config/dev_config.json

```diff
@@ -88,7 +88,8 @@
 		"rsvp": "http://localhost:8006/rsvp/internal/stats/",
 		"checkin": "http://localhost:8007/checkin/internal/stats/",
 		"user": "http://localhost:8003/user/internal/stats/",
-		"event": "http://localhost:8010/event/internal/stats/"
+		"event": "http://localhost:8010/event/internal/stats/",
+		"roles": "http://localhost:8002/auth/roles/internal/stats/"
```

Comment:
Based on the way I originally designed stats, this line should be `"auth": "http://localhost:8002/auth/internal/stats/"`. Each service should only have a single stats endpoint. If you need to have multiple sets of statistics for a service, that is fine, but you should embed those stats within seperate sub-json objects in the response from the single endpoint.